docs/cloud/README.md

```diff
@@ -1,5 +1,23 @@
 # Cloud Images
 
-At AlmaLinux, cloud integration is among our top priorities, ensuring that users get the most streamlined and efficient experience when deploying our OS in the cloud. Our dedication to this initiative is evident through the exceptional work of our Cloud Special Interest Group (Cloud SIG). They relentlessly work on perfecting and tailoring AlmaLinux for cloud environments, making our cloud images stand out for their performance, stability, and adaptability. 
+At AlmaLinux, cloud integration is among our top priorities, ensuring that users get the most streamlined and efficient experience when deploying our OS in the cloud. Our dedication to this initiative is evident through the exceptional work of our [Cloud Special Interest Group (Cloud SIG)](/sigs/Cloud). They relentlessly work on perfecting and tailoring AlmaLinux for cloud environments, making our cloud images stand out for their performance, stability, and adaptability. 
 
-Presently, we are proud to offer robust support for a wide range of platforms including AWS, Azure, Google Cloud, Oracle Cloud Infrastructure, OpenNebula, and Generic Cloud. Additionally, for those wishing to simulate cloud environments or test locally, we also provide options to run the Cloud image on your local machine. Dive in and experience the pinnacle of cloud-based Linux distributions with AlmaLinux.
+Presently, we are proud to offer robust support for a wide range of platforms including:
+| Cloud | AlmaLinux OS 8 | AlmaLinux OS 9 |
+| --- | --- | --- |
+| [AWS](AWS) | [Intel/AMD (x86_64)](https://aws.amazon.com/marketplace/pp/prodview-mku4y3g4sjrye)<br>[ARM64 (Aarch64)](https://aws.amazon.com/marketplace/pp/prodview-zgsymdwitnxmm) | [Intel/AMD (x86_64)](https://aws.amazon.com/marketplace/pp/prodview-ykmb6re2rcouy)<br>[ARM64 (Aarch64)](https://aws.amazon.com/marketplace/pp/prodview-pvttztrj2uzjk) | 
+| [Azure](Azure) | [Intel/AMD (x86_64)](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/almalinux.almalinux-x86_64?tab=Overview)<br>[ARM64 (Aarch64)](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/almalinux.almalinux-arm?tab=Overview)<br>[HPC](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/almalinux.almalinux-hpc?tab=Overview) | [Intel/AMD (x86_64)](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/almalinux.almalinux-x86_64?tab=Overview)<br>[ARM64 (Aarch64)](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/almalinux.almalinux-arm?tab=Overview) |
```

Comment:
Could you remove `?tab=Overview` endings from the URL of Azure Maketplace offerings?